PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/java/com/microsoft/azure/toolkit/intellij/azuresdk/IntelliJReferenceBookActionContributor.java

```diff
@@ -35,9 +35,11 @@ public class IntelliJReferenceBookActionContributor implements IActionsContribut
             put("Microsoft.ContainerRegistry", "Container Registry");
             put("Microsoft.ContainerRegistry/registries", "Container Registry");
             put("Microsoft.DBforMySQL", "Database for MySQL");
+            put("Microsoft.DBforMySQL/servers", "Database for MySQL");
```

Comment:
please add View Reference Book action to SQL as well

Reply:
fixed.